lib/lotus_notes/api.rb

```diff
@@ -0,0 +1,30 @@
+require 'mechanize'
+
+module LotusNotes
+  class Api
+
+    def self.get_person_events(params)
+      response_page = make_request(params)
+      person_events = JSON.parse(response_page.body)["events"]
```

Comment:
You don't need the assignment. Also, deal with errors and take into account when the page hasn't events (return empty array).